src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -55,7 +55,10 @@ const propTypes = {
         displayName: PropTypes.string,
 
         /** Avatar URL of the current user from their personal details */
-        avatar: PropTypes.string,
+        avatar: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+
+        /** Login email of the current user from their personal details */
```

Comment:
Don't think we need to say "from their personal details" since this is under `currentUserPersonalDetails`

```suggestion
        /** Login email of the current user */
```

Reply:
Also feel free to remove "from their personal details" from the previous 2 props (avatar & displayName)